### ISSUE TITLE
Increment/decrement buttons on expected number of students need labels

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#158]
In the course creator, the increment/decrement buttons don't have labels. Users with screen readers cannot tell what these buttons are for (although it's not hard to guess).

**Comment by ntdb** [comment]
Those buttons are rendered differently by individual browsers, the field is just an `<input type='number' />`. What would be the best way to remove confusion since labelling the buttons automatically added by browsers isn't possible?

**Comment by ragesoss** [comment]
Hmm... is it possible to add alt text? If not, we can just leave this be.

-Sage

**Comment by ntdb** [comment]
We have full control over the input but not the buttons rendered by the browser. Perhaps an (invisible) title on the input itself explaining its purpose?

**Comment by ragesoss** [comment]
Let's try that and I'll run it by Helaine. If that doesn't solve it, maybe I'll report it as a bug against her browser.

**Comment by ragesoss** [comment]
This works for her.
